CHANGELOG.md

```diff
@@ -19,21 +19,18 @@ The REST API versions in the table are just for your information as the values a
 
 ### Added
 
--   Pivot table totals can be turned on with menu. (RAIL-990)
-
--   Pivot table supports grouping of attribute cells when sorted by first attribute. (BB-1388, BB-1391)
-
--   Attribute filters can be specified by attribute element value (ONE-3610)
+-   Export insight into CSV or XLSX ([doc](https://sdk.gooddata.com/gooddata-ui/docs/on_export_ready.html))
+-   Pivot table supports grouping of attribute cells when sorted by first attribute ([doc](https://sdk.gooddata.com/gooddata-ui/docs/pivot_table_component.html#grouping))
+-   Pivot table totals can be turned on with menu ([doc](https://sdk.gooddata.com/gooddata-ui/docs/pivot_table_component.html#totals))
+-   Attribute filters can be specified by attribute element value (see `textFilter` in [filters doc](https://sdk.gooddata.com/gooddata-ui/docs/filter_visual_components.html) and [AFM doc](https://sdk.gooddata.com/gooddata-ui/docs/afm.html))
+-   Optional stacking ([doc](https://sdk.gooddata.com/gooddata-ui/docs/chart_config.html#configure-stacking))
```

Comment:
-   Insight export into CSV or XLSX ([doc](https://sdk.gooddata.com/gooddata-ui/docs/on_export_ready.html))

-   Support for attribute cell grouping in pivot tables when a table is sorted by first attribute ([doc](https://sdk.gooddata.com/gooddata-ui/docs/pivot_table_component.html#grouping))

-   Turning on/off totals in pivot tables via the totals menu ([doc](https://sdk.gooddata.com/gooddata-ui/docs/pivot_table_component.html#totals))

-   Specifying attribute filters by an attribute element value (see `textFilter` in [Filter Visual Components](https://sdk.gooddata.com/gooddata-ui/docs/filter_visual_components.html) and [Set Up an AFM Query](https://sdk.gooddata.com/gooddata-ui/docs/afm.html))

-   Optional stacking ([doc](https://sdk.gooddata.com/gooddata-ui/docs/chart_config.html#configure-stacking))

Reply:
Thanks. We've used your wording.